session_4/blog/app/controllers/articles_controller.rb

```diff
@@ -92,6 +91,30 @@ def destroy
     end
   end
 
+  def like
+    #to like an article
+    if @view
+      @view.update( liked: true )
+      msg = "success"
+    else
+      msg = "err"
+    end
+    likes = View.where( liked: true ).count
```

Comment:
Gives you likes for _all_ articles and not just the current article.

Reply:
Thanks for pointing out the mistake I will fix that.